recipes/pedcheck/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pedcheck" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/haraldgrove/{{ name }}/archive/master.zip
+  sha256: 239a9c31793cab0f4e4374ba01d174c018f836b41867870d493663c16f78fcc3
+  
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pedcheck
+   
+about:
+  home: https://watson.hgen.pitt.edu/register/docs/pedcheck.html
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
There is no ``license`` on this project. Could you confirm if it is really ``MIT`` license, please?